From 6b908687d37dc4e5c00df90058990da034506628 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@linaro.org>
Date: Tue, 9 Mar 2021 11:52:32 +0100
Subject: [PATCH 1/1] libckteec: add missing <unistd.h> for gid_t

Include missing unistd.h to support type gid_t. This change fixes [1].

Link: [1] http://autobuild.buildroot.net/results/34b9946e6d59112a7eead304933534ad4739a84c/build-end.log
Signed-off-by: Etienne Carriere <etienne.carriere@linaro.org>
Acked-by: Jerome Forissier <jerome@forissier.org>
[yann.morin.1998@free.fr: actual backport now that upstream applied it]
Signed-off-by: Yann E. MORIN <yann.morin.1998@free.fr>
---
 libckteec/src/invoke_ta.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libckteec/src/invoke_ta.c b/libckteec/src/invoke_ta.c
index 6236719..70575d0 100644
--- a/libckteec/src/invoke_ta.c
+++ b/libckteec/src/invoke_ta.c
@@ -18,6 +18,7 @@
 #include <sys/types.h>
 #include <tee_client_api.h>
 #include <teec_trace.h>
+#include <unistd.h>
 
 #include "ck_helpers.h"
 #include "invoke_ta.h"
-- 
2.25.1

